src/newSeed/stage3.ts

```diff
@@ -69,10 +75,11 @@ export class Stage3 extends BaseStage {
         message = `Please enter a number greater than zero for Category ${tokenDistrb.category} Amount`;
       } else if (!(tokenDistrb.lockup > 0)) {
         message = `Please enter a number greater than zero for Category ${tokenDistrb.category} Lock-up`;
+      } else {
+        totalDistribAmount = totalDistribAmount.add(tokenDistrb.amount);
       }
-      totalDistribAmount = totalDistribAmount.add(tokenDistrb.amount);
     });
-    if (!message && totalDistribAmount.gt(this.seedConfig.tokenDetails.maxSeedSupply)) {
+    if (!message && this.seedConfig.tokenDetails.maxSeedSupply && totalDistribAmount.gt(this.seedConfig.tokenDetails.maxSeedSupply)) {
```

Comment:
this whole block should be in an `else`.  then you'll already know that maxSeedSupply is ok and you don't need this check.